internal/cfgplugins/policyforwarding.go

```diff
@@ -366,6 +470,78 @@ func PolicyForwardingConfig(t *testing.T, dut *ondatra.DUTDevice, traffictype st
 	}
 }
 
+// PolicyForwardingConfigScale configures policy forwarding using multiple traffic-policies.
+// When the device does not support the OpenConfig path, vendor-specific CLI commands
+// are applied.
+func PolicyForwardingConfigScale(t *testing.T, dut *ondatra.DUTDevice, sb *gnmi.SetBatch, encapparams OCEncapsulationParams, pf *oc.NetworkInstance_PolicyForwarding, params OcPolicyForwardingParams) *gnmi.SetBatch {
+	t.Helper()
+
+	// Check if the DUT requires CLI-based configuration due to an OpenConfig deviation.
+	if deviations.PolicyForwardingOCUnsupported(dut) {
+		// If deviations exist, apply configuration using vendor-specific CLI commands.
+		switch dut.Vendor() {
+		case ondatra.ARISTA: // Currently supports Arista devices for CLI deviations.
+			var PolicyForwardingConfig strings.Builder
+
+			PolicyForwardingConfig.WriteString("traffic-policies\n")
+
+			for i := 1; i <= encapparams.Count; i++ {
+
+				PolicyForwardingConfig.WriteString(fmt.Sprintf(`
+    traffic-policy tp_cloud_id_3_%d
+    match setttlv6 ipv6
+       ttl 1
+       !
+       actions
+          count
+          redirect next-hop group 1V6_vlan_3_%d
+          set traffic class 3
+    !
+    match setttlv4 ipv4
+       ttl 1
+       !
+       actions
+          count
+          redirect next-hop group 1V4_vlan_3_%d
+          set traffic class 3
+    !
+    match ipv4-all-default ipv4
+       actions
+          count
+          redirect next-hop group 1V4_vlan_3_%d
+          set traffic class 3
+    !
+    match ipv6-all-default ipv6
+       actions
+          count
+          redirect next-hop group 1V6_vlan_3_%d
+          set traffic class 3
+    !`, i, i, i, i, i))
+			}
+
+			helpers.GnmiCLIConfig(t, dut, PolicyForwardingConfig.String())
+		default:
+			// Log a message if the vendor is not supported for this specific CLI deviation.
+			t.Logf("Unsupported vendor %s for native command support for deviation 'policy-forwarding config'", dut.Vendor())
+		}
+		return nil
+	} else {
+		var ruleSeq uint32 = 1
```

Comment:
https://google.github.io/styleguide/go/best-practices#initialization
> For consistency, prefer := over var when initializing a new variable with a non-zero value.


```suggestion
		ruleSeq := uint32(1)
```